test/unit/openscap_host_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_plugin_helper'
+
+class OpenscapHostTest < ActiveSupport::TestCase
+  setup do
+    disable_orchestration
+    User.current = users :admin
+    Setting[:token_duration] = 0
+    Scaptimony::Policy.any_instance.stubs(:ensure_needed_puppetclasses).returns(true)
+  end
+
+  test 'Host has policy' do
+    host = FactoryGirl.create(:host)
+    assert_empty(host.policies)
+    policy = FactoryGirl.create(:policy)
+
+    assert(policy.assign_hosts([host]), 'Host policies should be assigned')
+    assert(host.policies.include?(policy), 'Policy should be in Host policies')
```

Comment:
could you please use `assert_includes(collection, expected)`, it's more verbose and you don't have to specify custom message